docs/usermanual/userman_admin.rst

```diff
@@ -662,8 +662,10 @@ Most common settings
 ^^^^^^^^^^^^^^^^^^^^
 
 * DEBUG = <True or False> (default = False) -- Enables debug mode. In case you run into technical issues, enable this setting before troubleshooting / reporting.
-* CONTENT_ROOT = "<path to desired content folder>" (default=ka-lite/content)
+* CONTENT_ROOT = "<path to desired content folder>" (default=~/.kalite/content)
   This is the path that KA Lite will use to look for KA Lite video files to play.  Change the path to another local directory to get video files from that directory. NB! Directory has to be writable for the user running the server in order to download videos.
+* ASSESSMENT_ITEMS_DATABASE_PATH = "<path to desired content folder>" (default=~/.kalite/content/assessmentitems.sqlite)
```

Comment:
"<path to desired content folder>"? Probably better to have this be a locally derived setting, since otherwise things will break. e.g.: 
`* ASSESSMENT_ITEMS_DATABASE_PATH = os.path.join(CONTENT_ROOT, "assessment/khan/assessmentitems.sqlite")`

And what about the .version file? How is it managed?